tests/myst/signing-engine-test/.gitignore

```diff
@@ -0,0 +1,5 @@
+1
```

Comment:
These numbers appear unrelated?

Reply:
They will be files created by the test, added them here to avoid people pushing them in